src/pages/Rockets.js

```diff
@@ -0,0 +1,68 @@
+import { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import Rocket from './Rocket/Rocket';
+import { fetchRocket } from '../redux/Rockets/Rockets';
+
+const Rockets = () => {
+  const rocketsArray = useSelector((state) => state.rockectsReducer);
+  const dispatch = useDispatch();
+
+  const URL = 'https://api.spacexdata.com/v3/rockets';
+
+  const fetchData = async () => {
+    const response = await fetch(`${URL}`, {
+      method: 'GET',
+      headers: {
+        'Content-type': 'application/json; charset=UTF-8',
+      },
+    });
+    const jsonRespnse = await response.json();
+    // delete unwanted data
+    const wantedData = jsonRespnse.map((rocket) => (
+      {
+        id: rocket.rocket_id,
+        name: rocket.rocket_name,
+        image: rocket.flickr_images[0],
+        description: rocket.description,
+      }
+    ));
+    console.log(wantedData);
+    dispatch(fetchRocket(wantedData));
```

Comment:
- [ ] Kindly delete the console log statement. it looks ugly and it's not a good practice because sometimes it can stop execution and even cause security issues.